### DISCUSSION TITLE
UM, JULES and LFRic Apps Code Review Deadline

### DISCUSSION THREAD

**Discussion by jennyhickson** [MetOffice/simulation-systems#190] [Announcements]
Hello,

The code review deadline for changes wishing to be included in UM vn13.6, JULES vn7.6 or LFRic Apps vn1.1 is **31st May**. 

It is advised that tickets are with their SciTech reviewer by 24th May to give them time to finish reviews. 

Many Thanks,
Jenny, on behalf of the SSD team

**Comment by theabro** [comment]
Please note that due to the May half-term, there is reduced coverage to approve UKCA code and configuration changes in the week of the 27th-31st May. 

Please ensure that you have requested all UKCA approvals by the end of Thursday 23rd May to enable us to look at them in time.

Many thanks,
Luke Abraham, UKCA code owner
